content/paid_support.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Paid Support"
+date: 2021-10-08T18:55:30+01:00
+draft: false
+aliases:
+- /commercial_support.html
+---
+### **Professional help, directly from Rockstor developers & contributors**
+{{< center-this >}}
+
+---
+#### **Service currently unavailable: awaiting an 'Incident-based Support subscription' membership tier in our [Open Collective](https://opencollective.com/the-rockstor-project) non-profit.**
+---
+
+{{< /center-this >}}
+
+## Incident-based Support subscription
+
+A Rockstor 'Incident-based Support subscription' is a *"Support included"* variant of our 'Stable Updates subscription' membership tier.
+
+Primarily intended for installs´ subscribed-to/running our [Stable Channel](https://rockstor.com/docs/update-channels/update_channels.html#stable-channel) releases 'Built on' non EOL openSUSE Leap.
+
+Release Candidate (RC) [Testing Channel](https://rockstor.com/docs/update-channels/update_channels.html#testing-channel) installs may also be considered.
+
+Our [downloadable]({{< relref dls.md >}}) installers are Stable or Release Candidate status.
+
+Targeted response time 48 hours, see below for details.
+
+A complement to your existing IT resources. **We are here to help**
+
+{{< bootstrap-table table_class="table table-bordered border-primary table-striped" >}}
+
+ --- | Details
+ --- | ---
+**Valid duration:** | 1 year from date of purchase.
+**Support via:** | Support Email, [Zulip](https://rockstor.zulipchat.com), [Mastodon](https://mastodon.world/@TheRockstorProject) Direct messages, planned forum group/PM. 
+**Support Language:** | English, Portuguese to follow in time.
+**Response Time:** | 2 business days: cases prioritized by severity, we respond as quickly as possible.
+**Service Availability:** | 8 AM to 6 PM (Europe/Lisbon) during weekdays, excluding Portugal holidays.
+
+{{< /bootstrap-table >}}
+
+
+Before purchasing an 'Incident-based Support subscription', consider taking full advantage of our extensive [documentation](/docs/) and our friendly [community forum](https://forum.rockstor.com/).
+
+{{< button label="Documentation" link="/docs/" >}}
+{{< button label="Community Forum" link="https://forum.rockstor.com/" >}}
+
+---
+
+## Support FAQ
+{{< bootstrap-table table_class="table table-bordered border-primary table-striped" >}}
+
+Question | Answer
+--- | ---
+**How much do you charge for Rockstor support?** | Support is charged on a per incident basis, expected relaunch price €240 + tax (if applicable) per incident.
+**How is an incident defined?** | An incident is defined as a question relating to a specific, discrete issue, and may involve several interactions with support prior to that issue's resolution.
+**What is not an incident?** | If your issue is caused by a Rockstor package software bug then there is no charge. In this case we prioritising the related bugfix.
```

Comment:
**What is not an incident?** | If your issue is caused by a Rockstor package software bug then there is no charge. In this case we will be prioritising the related bugfix.